src/dodal/devices/i10/i10_apple2.py

```diff
@@ -196,17 +198,17 @@ def __init__(
         """
         super().__init__(name=name)
         with self.add_children_as_readables():
```

Comment:
```suggestion
```
I think we can remove this now.
I also have a question, what is the best way to have read  behave like it use to? For example in this case, when read is call it will read the readable in both id_ref and pgm_ref as well as i10_apple2.